### ISSUE TITLE
Galaxy

### ISSUE THREAD

**Issue by apexamodi** [greenelab/computational-reagents#12]
Galaxy is an online tool that can perform analyses on large genomic data sets. Various genomic tools are integrated into the platform and the parameters used to run each tool can be shared and saved allowing for greater reproducibility and transparency. 

The tool is helpful as an introduction to doing large scale genomic analyses, however you are limited by the capacity of the server in terms of how much data and how fast analyses can be run. Additionally parameters in certain tools available on Galaxy cannot be fully customized.

**Comment by oryoruk** [comment]
Heard of Galaxy before, but never used it myself. Playing around with it online, and asking a labmate who uses it: my impression is that the target audience is computationally less-savvy biologists. I have found it very useful as a hub for good genomics tools to try out.

Regarding reproducibility:
- In addition to parameters, I realized you can pick and record the version of the tool you use from a dropdown list.
- I think it can be helpful and make it easier to record parameters, analysis flows.


Here is their [Galaxy 101 page](https://github.com/nekrut/galaxy/wiki/Galaxy101-1), if anyone is interested:

**Comment by linzho** [comment]
I agree that Galaxy looks for the computationally less-savvy, and it looks like it'd be annoying to communicate a "pipeline" through Galaxy. That being said I think being able to save parameters and share them is a definitely strength.
